namesys/publisher.go

```diff
@@ -136,3 +137,31 @@ func ValidateIpnsRecord(k u.Key, val []byte) error {
 	}
 	return nil
 }
+
+// InitializeKeyspace sets the ipns record for the given key to
+// point to an empty directory.
+// TODO: this doesnt feel like it belongs here
```

Comment:
I'm really not sure... I put it three different places before it ended up here, and it still doesnt feel quite right.